packages/core/src/v3/waitUntil/index.ts

```
@@ -0,0 +1,54 @@
+  }
+}
+
+const NOOP_MANAGER = new NoopManager();
+
+export class WaitUntilAPI implements WaitUntilManager {
+  private static _instance?: WaitUntilAPI;
+
+  private constructor() {}
+
+  public static getInstance(): WaitUntilAPI {
+    if (!this._instance) {
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)